src/python/sympy_boilerplate.yaml

```diff
@@ -356,36 +356,49 @@ title: |
   Stirling number of the second kind
 code: |
   from sympy.functions.combinatorial.numbers import stirling
+  n = 6
+  k = 2
   stirling(n, k)
 ---
 title: |
   Stirling number of the first kind
 code: |
   from sympy.functions.combinatorial.numbers import stirling
+  n = 6
+  k = 2
   stirling(n, k, kind=1, signed=False)
 ---
 title: |
   Reduced Stirling number of the second kind
 code: |
   from sympy.functions.combinatorial.numbers import stirling
+  n = 6
+  k = 2
+  d = 3
```

Comment:
I feel like these n, k and d variables should be in the `setup: ...` at the very top, but well, those variables are already defined as symbols and used in other examples. So, I can imagine it makes sense to split this up into a new category, i.e. all `sympy.functions.combinatorial` with concrete numeric variable values in the setup.

I don't know how much work you want to do on this. I'm fine with merging this, and it can be enhanced as such a separate category later as well :-)

Reply:
I added a separate category for functions that work on numbers